packages/app-degree-pages/README.md

```diff
@@ -221,9 +221,9 @@ yarn jsdoc
 ```
 ## How to install
 
-1. Make sure you are set up to use the private npm registry at registry.web.asu.edu.
-<br/>See instructures in the 'How to use private package registry' here: [README.md](../../README.md)
-2. ```yarn add @asu/app-degree-pages```
+1. Either make sure you are part of the ASU github organization and follow the instructions [here](https://github.com/ASU/asu-unity-stack#-how-to-use-the-private-package-registry)  or, if you are  you can clone this repo and run `yarn install` and `yarn build` to build the package locally.
```

Comment:
Wouldn't have asked for this cleanup, but since I'd already flagged another fix, might as well get nit-picky: Extra spaces in `)  or, if you are  you` - repeated in a few places in this PR.